projects/smart-table/src/lib/table-input-filter/table-input-filter.component.ts

```diff
@@ -1,36 +1,20 @@
-import { FilterComponent } from '@acpaas-ui/ngx-components/utils';
-import { Component, Input, OnInit } from '@angular/core';
+import {FilterComponent} from '@acpaas-ui/ngx-components/utils';
+import {Component, OnInit} from '@angular/core';
 import _ from 'lodash';
 
-import { AbstractFilter } from '../filter/abstract-filter';
+import {AbstractFilter} from '../filter/abstract-filter';
+import {SmartTableFilter} from '../filter/filter.decorator';
 
 @Component({
   selector: 'aui-table-input-filter',
-  templateUrl: './table-input-filter.component.html'
+  templateUrl: './table-input-filter.component.html',
+  styleUrls: ['../filter/filter.component.scss']
```

Comment:
Since you're using this twice, wouldn't it be better to just put in into the global styles?